src/v2/guide/list.md

```diff
@@ -231,9 +231,9 @@ However, this won't automatically pass any data to the component, because compon
 </my-component>
 ```
 
-The reason for not automatically injecting `item` into the component is because that makes the component tightly coupled to how `v-for` works. Being explicit about where its data comes from makes the component reusable in other situations.
+La raison pour ne pas injecter automatiquement `item` dans le composant est que : cela forcerait le composant à être fortement couplé avec l'utilisation de `v-for`. En étant explicite sur la manière dont les données sont injectées nous permettons au composant d'être réutilisable dans d'autres situations.
```

Comment:
Les deux points ou points virgule était nécessaire pour justifier, en français, l'utilisation double du mot composant. Je l'ai viré et j'ai viré sa seconde itération.